operators/multiclusterobservability/pkg/rendering/renderer_mcoa_test.go

```diff
@@ -156,7 +169,20 @@ func TestRenderAddonDeploymentConfig(t *testing.T) {
 			},
 		},
 	}
-	renderer := &MCORenderer{cr: mco}
+
+	config.SetMonitoringCRName("multicluster-observability")
+	scheme := runtime.NewScheme()
+	assert.NoError(t, mcov1beta2.AddToScheme(scheme))
+	fakeClient := fake.NewClientBuilder().
+		WithScheme(scheme).
+		WithObjects(mco).
+		Build()
+
+	renderer := &MCORenderer{cr: mco, kubeClient: fakeClient, rendererOptions: &RendererOptions{
+		MCOAOptions: MCOARendererOptions{
+			MetricsHubHostname: "observability-hub",
+		},
+	}}
```

Comment:
We can simplify here
```suggestion
	renderer := &MCORenderer{cr: mco, rendererOptions: &RendererOptions{
		MCOAOptions: MCOARendererOptions{
			MetricsHubHostname: "observability-hub",
		},
	}}
```